front_end/src/reducers/bookingReducer.js

```diff
@@ -0,0 +1,41 @@
+import { SAVE_BOOKING, SET_BOOKING } from "../actions/types";
+
+const initialState = {
+  tempBookingData: null,
+  bookingConfirmationData: null
+  // bookingId: "test",
```

Comment:
I don't know if this is unneeded code or it will be needed in the future.

Reply:
Minh asked me to leave it in as a guideline to what we will need from the backend....